coc.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Code of Conduct
+categories: nav
+showinnav: true
+---
+
+# {{ page.title }}
+
+## Austin Hackers Association (AHA!) Code of Conduct
+
+Welcome to AHA! If you’re here, you already know this isn’t your typical buttoned-up tech meetup. We swear, we heckle, and we’re all just here to have a great fucking time. But even in our chaotic little corner, we have some ground rules.
+
+#### Roast the Ideas, Not the Person
+
+>> Got a hot take? Bring it. Got a dumb take? Expect to hear about it. But keep it fun. Don’t be a dick to the person; they’re here to learn and blow off steam just like you.
```

Comment:
nit: I think these render slightly better if it's just a single `>` or maybe none at all but no strong feelings about it

Reply:
It looks like trash and I will happily accept any and all format changes.

(I'm actually working on the #148 ticket for this reason)